docs/docs/getting-started/first-steps-local.md

```diff
@@ -2,32 +2,101 @@
 
-## Prerequisites
+## Quick setup
 
-* A Linux OS with the following components installed
-  * [Constellation CLI](./install.md#install-the-constellation-cli)
-  * [KVM kernel module](https://www.linux-kvm.org/page/Main_Page)
-  * [Docker](https://docs.docker.com/engine/install/)
-  * [xsltproc](https://gitlab.gnome.org/GNOME/libxslt/-/wikis/home)
-  * (Optional) [virsh](https://www.libvirt.org/manpages/virsh.html) to observe and access your nodes
-* Other system requirements
+### Prerequisites
+
+* [Install Terraform](https://developer.hashicorp.com/terraform/downloads)
+* [Install the Azure CLI](https://learn.microsoft.com/de-de/cli/azure/install-azure-cli)
+  to authenticate with Azure:
+
+  ```sh
+  az login
+  ```
+
+### Instructions
+
+Through the Terraform template for Azure it's very easy to set up a MiniConstellation cluster on a remote VM.
+
+1. Clone the Constellation repository:
+
+    ```sh
+    git clone https://github.com/edgelesssys/constellation.git
+    ```
+
+2. Set up the remote Azure VM through Terraform:
+
+    By default, the [`Standard_D8s_v5`](https://learn.microsoft.com/de-de/azure/virtual-machines/dv5-dsv5-series) machine type is selected which supports nested virtualization, but you can also set another supported machine type in Terraform (`machine_type`) by referring to the [Azure docs](https://azure.microsoft.com/en-us/blog/nested-virtualization-in-azure/).
+    Then run:
+
+    ```sh
+    cd constellation/dev-docs/miniconstellation/azure-terraform
+    ./create-vm.sh
+    ```
+
+    After execution, you should be connected with the remote machine through SSH.
+
+3. Prepare the VM for `constellation mini up`
+
+   For convenience, you will find a script on the machine that installs the Constellation CLI, makes sure that the Docker service is running and executes `constellation mini up`:
+
+   ```sh
+   ./setup-miniconstellation.sh
+   ```
+
+    The cluster creation takes about 15 minutes.
+4. Clean up cloud resources
+
+    Exit the SSH connection and run:
+
+    ```sh
+    terraform destroy
+    ```
+
+## Manual setup
+
+### Prerequisites
+
+* machine requirements:
   * An x86-64 CPU with at least 4 cores (6 cores are recommended)
   * At least 4 GB RAM (6 GB are recommended)
   * 20 GB of free disk space
-  * Hardware virtualization enabled in the BIOS/UEFI (often referred to as Intel VT-x or AMD-V/SVM)
-  * `iptables` rules configured to not drop forwarded packages.
-    If running the following command returns no error, please follow [the troubleshooting guide](#vms-have-no-internet-access):
+  * Hardware virtualization enabled in the BIOS/UEFI (often referred to as Intel VT-x or AMD-V/SVM) / nested-virtualization support when using a VM
+* OS / library requirements:
+  * recommended: Ubuntu 22.04 LTS
```

Comment:
Are those requirements below automatically installed with Ubuntu 22.04?

Reply:
I agree it's confusing. Maybe we just limit ourselves to supporting Ubuntu and list all packages that are required (see dev-docs/miniconstellation/azure-terraform/cloud-init.yaml)?

Reply:
I agree that the part about using constellation in `setup-miniconstellation.sh` should not be hidden. So I put that in the  first-steps and just refer to the script as a quick way?